frontend/src/metabase/components/FieldValuesWidget.jsx

```diff
@@ -291,6 +296,7 @@ class FieldValuesWidgetInner extends Component {
 
     const tokenFieldPlaceholder = getTokenFieldPlaceholder({
       fields,
+      parameter,
```

Comment:
This is the first place where we didn't take parameter into account, basically when there's a dashboard filter (parameter) set, we should use that filter's type instead of field types.